src/styles/blocks/card.scss

```diff
@@ -0,0 +1,113 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  box-sizing: border-box;
+  width: 200px;
+  height: 408px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &:hover {
+    transition-property: 300ms;
```

Comment:
The `transition-property` should be `transition` to include both `transform` and `color` properties for smooth transitions. Use `transition: transform 300ms, color 300ms;` to achieve the required effect.